### PR TITLE
Burger perk lets hope this owrks

### PR DESCRIPTION
**Pull request by mhuang306**

### CODE REVIEW

File: Assets/Code/Scripts/Perks/Burger.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class Burger : Perk
+{
+
+    public int CapacityIncrease = 50;
+    public int Heal = 20;
+    public float SizeIncrease = .4f;
```
Comment (anjx123) [inline Assets/Code/Scripts/Perks/Burger.cs:11]: Fields should be private and serialized. Field names should not be capitalized. CapacityIncrease could be changed to maxHealthIncrease.

File: Assets/Code/Scripts/Perks/Burger.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class Burger : Perk
+{
+
+    public int CapacityIncrease = 50;
+    public int Heal = 20;
```
Comment (anjx123) [inline Assets/Code/Scripts/Perks/Burger.cs:10]: Is there a reason that Heal doesn't match capacityIncrease? This means that a player will have 150 maxHealth, but only have 120 starting health

File: Assets/Code/Scripts/Perks/Burger.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    void Start()
+    {
+        perkName = "Burger";
+        description = "Increase health capacity by: "+ CapacityIncrease + ", heal " + Heal +"HP and " + "increase size by " + SizeIncrease * 100 + "percent";
+
+    }
+
+    public override void OnStart(Player player)
+    {
+        player.GetComponent<PlayerHealth>().maxHealth += CapacityIncrease;
```
Comment (anjx123) [inline Assets/Code/Scripts/Perks/Burger.cs:22]: Use Player's GetHealth method

File: Assets/Code/Scripts/Perks/Burger.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        player.GetComponent<PlayerHealth>().TakeDamage(-Heal);
```
Comment (anjx123) [inline Assets/Code/Scripts/Perks/Burger.cs:23]: If it is unintended that the player has to heal after increasing their maxHealth. Use the player.GetHealth().MaxHealth property, it takes care of healing after increasing the maxHealth so you dont have to worry about it.

**Review by anjx123 — CHANGES_REQUESTED** [review]

**Review by Hanson-Sun — APPROVED** [review]
lgtm

**Review by anjx123 — APPROVED** [review]
lgtm

**Pull request merged**